src/bindings/c/ov/src/CMakeLists.txt

```diff
@@ -0,0 +1,50 @@
+# Copyright (C) 2018-2022 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+#
+
+set(TARGET_NAME openvino_ov_c)
+
+file(GLOB SOURCES ${CMAKE_CURRENT_SOURCE_DIR}/*.cpp)
+file(GLOB HEADERS ${InferenceEngine_C_API_SOURCE_DIR}/include/*.h)
+
+# create library
+
+add_library(${TARGET_NAME} ${HEADERS} ${SOURCES})
+add_library(openvino::runtime::ov ALIAS ${TARGET_NAME})
+
+target_link_libraries(${TARGET_NAME} PRIVATE openvino)
+
+target_include_directories(${TARGET_NAME} PUBLIC
+    $<BUILD_INTERFACE:${InferenceEngine_C_API_SOURCE_DIR}/include>)
+
+if(NOT BUILD_SHARED_LIBS)
+    target_compile_definitions(${TARGET_NAME} PUBLIC OPENVINO_STATIC_LIBRARY)
+endif()
+
+add_cpplint_target(${TARGET_NAME}_cpplint FOR_TARGETS ${TARGET_NAME})
```

Comment:
Please enable clang_format instead of cpplint

Reply:
ok, we will submit a PR to do it.